src/shapes/group.class.js

```diff
@@ -195,6 +219,9 @@
     _onObjectAdded: function(object) {
       this.dirty = true;
       object.group = this;
+      if (!object.canvas && this.triggerObjectsEvents.indexOf('added') > -1) {
+        object.fire('added');
+      }
```

Comment:
this is a different fix i guess, and i bet is missing the remove counterpart?

Reply:
would make sense that each object that gets added either to canvas or to group fire added. For canvas happens, for groups not.

Reply:
No actually this is the same fix, it’s just that I was looking for a common « add » method that would also be called on group initialization if an array of objects is passed as first argument but realized it wasn’t the case. So, basically, if objects are passed in initialization, the added event will get fired as soon as the group is added, but if you then manually add another object and this object wasn’t already on the canvas, it will also trigger the added event as expected

Reply:
Oh and no I didn’t add the removed counterpart because if I understand correctly, if an object gets removed from the group it doesn’t necessarily mean that it gets removed from the canvas? Is that correct?